<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CounterpartyForm</class>
 <widget class="QDialog" name="CounterpartyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 254, 178);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>50</y>
     <width>354</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="leName">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBoxType">
      <property name="styleSheet">
       <string notr="true">background-color:white; border: 1px solid black; border-radius: 0px; padding: 1px;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbEmail">
      <property name="text">
       <string>Почта</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbUsed">
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbType">
      <property name="text">
       <string>Тип</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbAddress">
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:green;
font-size: 36px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Контрагент&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbName">
      <property name="text">
       <string>Название</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="teAddress">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lePhoneNumber">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="leEmail">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:green;
font-size: 36px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Добавление контрагента&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>298</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="2">
     <widget class="QPushButton" name="delete_2">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="save">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="add">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="previous">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="logout">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="next">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Вперед</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="cancel">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>306</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Поиск по названию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetMap" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openMail">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightgreen</string>
   </property>
   <property name="text">
    <string>Написать письмо</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openMapButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>441</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightgreen</string>
   </property>
   <property name="text">
    <string>Открыть карту в браузере</string>
   </property>
  </widget>
  <widget class="QPushButton" name="report">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightgreen</string>
   </property>
   <property name="text">
    <string>Открыть отчёт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
